creating-overview.Rmd

```diff
@@ -173,13 +173,13 @@ unique(purrr::map_chr(b$x$data, "type"))
 #> [1] "histogram"
 ```
 
-Here we've learned that **plotly** creates 8 histogram traces to generate the dodged bar chart: one trace for each level of `clarity`.^[Although the x-axis is discrete, plotly.js still considers this a histogram because it's generating it generates counts in the browser. Learn more about the difference between histograms and bar charts in section \@ref(bars-histograms).] Why one trace per category? As illustrated in Figure \@ref(fig:intro-show-hide) there are two main reasons: to populate a tooltip and legend entry for each level of `clarity` level.
+Here we've learned that **plotly** creates 8 histogram traces to generate the dodged bar chart: one trace for each level of `clarity`.^[Although the x-axis is discrete, plotly.js still considers this a histogram because it's generating it generates counts in the browser. Learn more about the difference between histograms and bar charts in section \@ref(bars-histograms).] Why one trace per category? As illustrated in Figure \@ref(fig:intro-show-hide), there are two main reasons: to populate a tooltip and legend entry for each level of `clarity` level.
 
```

Comment:
Should be `section \@ref(bars-histograms)`?

Reply:
Yes, I think it should be like this to show that you are referring to another chapter. 

You could keep the reference to a section like “In Section 12.3”. This will be similar to the reference to figures.